pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -91,6 +92,12 @@ func ToWorkflowTrigger(obj interface{}) (*v1alpha1.WorkflowTrigger, error) {
 func (c *CronTrigger) Run() {
 
 	c.WorkflowRun.Name = c.WorkflowTriggerName + "-" + strings.Replace(uuid.NewV1().String(), "-", "", -1)
+
+	if c.WorkflowRun.Labels == nil {
+		c.WorkflowRun.Labels = make(map[string]string)
+	}
+	c.WorkflowRun.Labels[CleverWorkflowName] = c.WorkflowRun.Spec.WorkflowRef.Name
```

Comment:
Please use annotations to add workflow name to WorkflowRun.

Reply:
no, my leader asked me to use `label` instead of annotations

@codeflitting